Marlin/src/libs/stopwatch.cpp

```diff
@@ -38,6 +42,7 @@ bool Stopwatch::stop() {
 
   if (isRunning() || isPaused()) {
     TERN_(EXTENSIBLE_UI, ExtUI::onPrintTimerStopped());
+    TERN_(IS_DWIN_MARLINUI, ui.did_first_redraw = false);
```

Comment:
The `did_first_redraw` flag is meant to be internal and private, not something that is meant for high-level code to hack. If the screen needs to redraw differently due to a change in paused / active printing state, the screen implementation ought to handle this internally. There are no other MarlinUI-based controllers that need or implement a special handler for `onPrintTimerStarted`/`onPrintTimerPaused`/`onPrintTimerStopped` but that's the method that perhaps should be added, with an empty implementation for all screens except this one.

Reply:
OK. My initial thought was to track the state internally every time the status screen was drawn and update `did_first_redraw` as needed, but this occurred to me as a way of not having that check on ever draw, but I can see wanting to keep that internal too.

I'll switch it back to that initial approach.